v3.3.2 release notes (#12)
ocient/pyocient_release: README.md

# Pyocient Release Notes

## 3.3.2

> - Fix issue with ending SSO sessions when closing connections

Reviewer: ```suggestion
- Fix ending SSO sessions when you close the connection.
```
